src/screens/Movie/Movie.tsx

```diff
@@ -115,56 +134,51 @@ const Movie = (): JSX.Element => {
         ))}
         {item ? <script type="application/ld+json">{generateMovieJSONLD(item)}</script> : null}
       </Helmet>
-      <Cinema
-        open={play && isEntitled}
-        onClose={goBack}
+      <VideoLayout
         item={item}
-        title={item.title}
-        primaryMetadata={primaryMetadata}
-        onComplete={handleComplete}
-        feedId={feedId ?? undefined}
-      />
-      <TrailerModal item={trailerItem} title={`${item.title} - Trailer`} open={playTrailer} onClose={() => setPlayTrailer(false)} />
-      <VideoDetails
         title={item.title}
         description={item.description}
+        inlineLayout={inlineLayout}
         primaryMetadata={primaryMetadata}
-        image={item.backgroundImage}
```

Comment:
Is `image` missing in the `VideoDetails` component? I also don't see an image anymore :-(

Reply:
Fixed